src/components/header.js

```diff
@@ -1,5 +1,113 @@
-import React from 'react'
+import React, { useState } from 'react'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import Typography from '@material-ui/core/Typography'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import {
+  makeStyles,
+  createMuiTheme,
+  ThemeProvider,
+  Avatar,
+  Grid,
+} from '@material-ui/core'
 
-const Header = () => <div>Header</div>
+const transparentTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#E5E5E5', contrastText: '#ffffff' },
+  },
+})
 
-export default Header
+const blackTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#000000' },
+  },
+})
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  header: {
+    margin: 'auto',
```

Comment:
add a background property and set that to 'transparent'
`background: 'transparent'`